device/bgp/bgpsrtebindingsubtlv.yaml

```diff
@@ -2,12 +2,14 @@ components:
   schemas:
     BgpSrte.BindingSubTlv:
       description: >-
-        Configuration for BGP binding sub TLV settings.
+        Configuration for the binding SID sub-TLV. 
+        This is used to signal the binding SID related information 
+        of the SR Policy candidate path.
       type: object
       properties:
         binding_sid_type:
           description: >-
-            TBD
+            Specify one of the No Binding SID, Four Octets Sid or IPv6 SID.
```

Comment:
should be more readable like one of {No Binding SID, Four Octets Sid, IPv6 SID}

Reply:
taken care